modules/jaeger-config-storage.adoc

```diff
@@ -0,0 +1,207 @@
+
+[CAUTION]
+====
+There can be only one Jaeger with self-provisioned Elasticsearch instance per namespace.
+====
+
+.Jaeger storage parameters
+[options="header"]
+[cols="l, a, a, a"]
+|===
+|Parameter |Description |Values |Default value
+
+
+|spec:
+ storage:
+  options: {}
+|Configuration options that define the storage.
+|
+|
+
+|storage:
+  type:
+|Type of storage to use for the deployment.
+|`memory` or `elasticsearch`
+|`memory`
+
+|storage:
+  options:
+   es:
+    server-urls:
+|URL of the Elasticsearch instance.
+|The fully-qualified domain name of the Elasticsearch server.
+|`http://elasticsearch.<namespace>.svc:9200`
+
+|elasticsearch:
+  nodeCount:
+|Number of Elasticsearch nodes. For high availability use at least 3 nodes. Do not use 2 nodes as “split brain” problem can happen.
+|Integer value.  For example, Proof of concept = 1,
+Minimum deployment =3
+|1
+
+|elasticsearch:
+  redundancyPolicy:
+|Controls number of Elasticsearch replica shards.
+|`ZeroRedundancy`(no replica shards), `SingleRedundancy`(one replica shard), `MultipleRedundancy`(each index is spread over half of the Data nodes), `FullRedundancy` (each index is fully replicated on every Data node in the cluster)
+|
+
+|elasticsearch:
+  resources:
+    requests:
+      cpu:
+|Number of central processing units for requests, based on your environment’s configuration.
+|Specified in cores or millicores (for example, 200m, 0.5, 1).  For example, Proof of concept = 500m,
+Minimum deployment =1
+|1Gi
+
+|elasticsearch:
+  resources:
+    requests:
+      memory:
+|Available memory for requests, based on your environment’s configuration.
+|Specified in bytes (for example, 200Ki, 50Mi, 5Gi). For example, Proof of concept = 1Gi,
+Minimum deployment = 16Gi*
+|500m
+
+|elasticsearch:
+  resources:
+    limits:
+      cpu:
+|Limit on number of central processing units, based on your environment’s configuration.
+|Specified in cores or millicores (for example, 200m, 0.5, 1). For example, Proof of concept = 500m,
+Minimum deployment =1
+|
+
+|elasticsearch:
+  resources:
+    limits:
+      memory:
+|Available memory limit based on your environment’s configuration.
+|Specified in bytes (for example, 200Ki, 50Mi, 5Gi). For example, Proof of concept = 1Gi,
+Minimum deployment = 16Gi*
+|
+
+|elasticsearch:
+  secretname:
+|Name of the secret, for example `jaeger-secret`.
+|
+|N/A
+
+
+|storage:
+  esIndexCleaner:
+    enabled:
+|When using Elasticsearch storage, by default a job is created to clean old traces from the index.  This parameter enables or disables the index cleaner job.
+|`true`/ `false`
+|`true`
+
+|storage:
+ esIndexCleaner:
```

Comment:
Why there is `esIndexCleaner` node three times? The other ones are on line 103 and 117